### PR TITLE
Sm/noun first

### PR DESCRIPTION
**Pull request by mshanemc**
commands are changing names again

`generate:cmdt:x` => `cmdt:generate:x`

No aliases are required because tax-free handles this.

@W-12436084@

### CODE REVIEW

File: messages/fromorg.md [hunk 1]
```diff
@@ -4,35 +4,35 @@ Generate a custom metadata type and all its records from a Salesforce object.
...
+Use this command to migrate existing custom objects or custom settings in an org to custom metadata types. If a field of the Salesforce object is of an unsupported type, the field type is automatically converted to text. Run "<%= config.bin %> cmdt generate field --help" to see the list of supported cmdt field types, listed in the --type flag summary. Use the --ignore-unsupported to ignore these fields.
 
-This command creates the metadata files that describe the new custom metadata type and its fields in the "force-app/main/default/objects/TypeName__mdt" directory by default, where "TypeName" is the value of the required --dev-name flag. Use --type-output-directory to create them in a different directory.
+This command creates the metadata files that describe the new custom metadata type and its fields in the "force-app/main/default/objects/TypeName\_\_mdt" directory by default, where "TypeName" is the value of the required --dev-name flag. Use --type-output-directory to create them in a different directory.
```
Comment (jshackell-sfdc) [inline messages/fromorg.md:9]:
```suggestion
This command creates the metadata files that describe the new custom metadata type and its fields in the "force-app/main/default/objects/TypeName__mdt" directory by default, where "TypeName" is the value of the required --dev-name flag. Use --type-output-directory to create them in a different directory.
```

File: messages/fromorg.md [hunk 2]
```diff
@@ -4,35 +4,35 @@ Generate a custom metadata type and all its records from a Salesforce object.
...
 
 # examples
 
-- Generate a custom metadata type from a custom object called MySourceObject__c in your default org:
+- Generate a custom metadata type from a custom object called MySourceObject\_\_c in your default org:
```
Comment (jshackell-sfdc) [inline messages/fromorg.md:13]:
```suggestion
- Generate a custom metadata type from a custom object called MySourceObject__c in your default org:
```

File: messages/fromorg.md [hunk 3]
```diff
@@ -4,35 +4,35 @@ Generate a custom metadata type and all its records from a Salesforce object.
...
 
-  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject__c
+  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject\_\_c
```
Comment (jshackell-sfdc) [inline messages/fromorg.md:15]:
```suggestion
  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject__c
```

File: messages/fromorg.md [hunk 4]
```diff
@@ -4,35 +4,35 @@ Generate a custom metadata type and all its records from a Salesforce object.
...
 
 - Generate a custom metadata type from a custom object in an org with alias my-scratch-org; ignore unsupported field types instead of converting them to text:
 
-  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject__c --ignore-unsupported --target-org my-scratch-org
+  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject\_\_c --ignore-unsupported --target-org my-scratch-org
```
Comment (jshackell-sfdc) [inline messages/fromorg.md:19]:
```suggestion
  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject__c --ignore-unsupported --target-org my-scratch-org
```

File: messages/fromorg.md [hunk 5]
```diff
@@ -4,35 +4,35 @@ Generate a custom metadata type and all its records from a Salesforce object.
...
 
 - Generate a protected custom metadata type from a custom object:
 
-  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject__c --visibility Protected
+  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject\_\_c --visibility Protected
```
Comment (jshackell-sfdc) [inline messages/fromorg.md:23]:
```suggestion
  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject__c --visibility Protected
```

File: messages/fromorg.md [hunk 6]
```diff
@@ -4,35 +4,35 @@ Generate a custom metadata type and all its records from a Salesforce object.
...
 
 - Generate a protected custom metadata type from a custom setting with a specific singular and plural label:
 
-  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --label "My CMDT" --plural-label "My CMDTs" --sobject MySourceSetting__c --visibility Protected
+  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --label "My CMDT" --plural-label "My CMDTs" --sobject MySourceSetting\_\_c --visibility Protected
```
Comment (jshackell-sfdc) [inline messages/fromorg.md:27]:
```suggestion
  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --label "My CMDT" --plural-label "My CMDTs" --sobject MySourceSetting__c --visibility Protected
```

File: messages/fromorg.md [hunk 7]
```diff
@@ -4,35 +4,35 @@ Generate a custom metadata type and all its records from a Salesforce object.
...
 
 - Generate a custom metadata type and put the resulting metadata files in the specified directory:
 
-  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject__c --type-output-directory path/to/my/cmdt/directory
+  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject\_\_c --type-output-directory path/to/my/cmdt/directory
```
Comment (jshackell-sfdc) [inline messages/fromorg.md:31]:
```suggestion
  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject__c --type-output-directory path/to/my/cmdt/directory
```

File: messages/fromorg.md [hunk 8]
```diff
@@ -4,35 +4,35 @@ Generate a custom metadata type and all its records from a Salesforce object.
...
 
 - Generate a custom metadata type and put the resulting record metadata file(s) in the specified directory:
 
-  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject__c --records-output-dir path/to/my/cmdt/record/directory
+  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject\_\_c --records-output-dir path/to/my/cmdt/record/directory
```
Comment (jshackell-sfdc) [inline messages/fromorg.md:35]:
```suggestion
  <%= config.bin %> <%= command.id %> --dev-name MyCMDT --sobject MySourceObject__c --records-output-dir path/to/my/cmdt/record/directory
```

**Review by jshackell-sfdc — CHANGES_REQUESTED** [review]
Errant backslashes are back, so I removed 'em.

**Review by iowillhoit — APPROVED** [review]

**Pull request merged**